config/src/main/scala-2.12/main/scala/io.circe.config/compat/package.scala

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I think we have an extra main/scala in the dir structure of this and the scala-2.13 dir.

Reply:
Well spotted! Dropped in https://github.com/circe/circe-config/pull/361/commits/0d5bed576b84c0a866be4d5ca33a0343fe0ad8ff.